app/components/ui/molecules/PeoplePicker.test.js

```diff
@@ -163,5 +163,12 @@ describe('PeoplePicker', () => {
       searchFor('s')
       expect(searchWrapper.find('PersonPod')).toHaveLength(0)
     })
+
+    it('filters the people pods when you click on the search icon', () => {
+      const input = searchWrapper.find('input')
+      input.simulate('change', { target: { value: 'annie' } })
+      searchWrapper.find('[data-test-id="search-icon"]').simulate('click')
+      expect(searchWrapper.find('PersonPod').length).toBeGreaterThan(0)
```

Comment:
This just checks whether any results are displayed at all, no?

Wouldn't this test pass, even if the results hadn't been filtered or changed at all, so long as there were still person pods in the grid?

Reply:
Yeah, totally forgot all person pods are displayed by default. I'll change it.